app.js

```diff
@@ -138,9 +148,16 @@ sitemap(config, (sitemap, urls) => {
 				if (!!error.pageNotFound && !!record) {
 					pages.deleteObject(record.objectID, (error, result) => {
 						console.log('%d - Deleted %s:%s (%s)', id, record.objectID, record.lang, record.url);
+
+						//Ping back delete
+						pingbackUrl({
+							id, id,
+							result: 'success',
+							action: 'delete',
+							url: url.url
+						}, tearDown);
```

Comment:
Why not pass the callback in the data object ?

Reply:
And why wait for the pingback to complete ?

Reply:
To keep console log in sync.

Reply:
ah ok